From 92796b1bdc2056eeda5c571b86efc5f7c2164726 Mon Sep 17 00:00:00 2001
From: Sven Seeberg <mail@sven-seeberg.de>
Date: Wed, 30 Jan 2019 16:11:48 +0100
Subject: [PATCH 3/3] Add Integreat labels to tree view

* Add function call for Integreat lables in
  the CMS tree view.
---
 wp-content/plugins/cms-tree-page-view/functions.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wp-content/plugins/cms-tree-page-view/functions.php b/wp-content/plugins/cms-tree-page-view/functions.php
index af938c23..771f446d 100755
--- a/wp-content/plugins/cms-tree-page-view/functions.php
+++ b/wp-content/plugins/cms-tree-page-view/functions.php
@@ -1504,7 +1504,7 @@ function cms_tpv_print_childs($pageID, $view = "all", $arrOpenChilds = null, $po
 					"id": "cms-tpv-<?php echo $onePage->ID ?>",
 					"post_id": "<?php echo $onePage->ID ?>",
 					"post_type": "<?php echo $onePage->post_type ?>",
-					"post_status": "<?php echo $onePage->post_status ?>",
+					"post_status": "<?php echo ig_tree_view_labels ($onePage->ID, $onePage->post_status ) ?>",
 					"post_status_translated": "<?php echo isset($post_statuses[$onePage->post_status]) ? $post_statuses[$onePage->post_status] : $onePage->post_status  ?>",
 					"rel": "<?php echo $rel ?>",
 					"childCount": <?php echo ( !empty( $arrChildPages ) ) ? sizeof( $arrChildPages ) : 0 ; ?>,
-- 
2.16.4

